stac_fastapi/api/stac_fastapi/api/config.py

```diff
@@ -22,3 +28,18 @@ class AddOns(enum.Enum):
     """Enumeration of available third party add ons."""
 
     bulk_transaction = "bulk-transaction"
+
+
+class Settings(BaseSettings):
```

Comment:
The fastapi app that comes packaged in stac-fastapi isn't necessary to use the project, and in the past has been talked about as a convenience rather than a core feature of the library (given that a client project perhaps should create their own App to customize with any middleware particular to the application, etc).

Since these settings apply to the fastapi app, perhaps we should give them a name that's less generic - e.g. a reader might thing that these are the most core settings, vs "ApiSettings".

Perhaps `FastApiAppSettings`?

Same with the exported instance below.

Reply:
@lossyrob thanks for the good suggestion. I've been unable to look at this for a while but thanks to @moradology this has been implemented